### DISCUSSION TITLE
any tips for tracking storms? (Historical temporal data for Earth)

### DISCUSSION THREAD

**Discussion by SriSathyaSaiVidyaVahini** [OpenSpace/OpenSpace#3183] [Q&A] [answered]

**Comment by hn-88** [comment]
Hi, I thought of creating some fulldome content showing the progress of various severe storms like [Cyclone Phailin](https://en.wikipedia.org/wiki/Cyclone_Phailin) or [Katrina](https://en.wikipedia.org/wiki/Hurricane_Katrina). But it looks like the default profile's temporal data is only available for a few months? I could see Storm Kathleen over the UK (April 10 2024), but couldn't see Storm Michaung over India (Dec 5 2023).
1.Any tips on whether older data can be easily shown by enabling or creating a data later for Earth? Or if [creating a new renderable globe like this](https://docs.openspaceproject.com/en/latest/manual/content/globebrowsing/creation/creating-a-renderableglobe.html) is the only way?
2. At present I can think of advancing time by one day and taking snapshots. Any better ideas?

**Comment by alexanderbock** [comment, marked as answer]
The VIIRS layer that is enabled by default should work for December 5th as well.  There might be issues if you are caching that layer and you have accessed the dates before, but clearing the cache should fix that.  One simple way to check the data availability is to use [WorldView](https://worldview.earthdata.nasa.gov/?v=-10.955847428174224,-32.33245079259604,163.99364284059314,67.23004920740394&l=Reference_Labels_15m(hidden),Reference_Features_15m(hidden),Coastlines_15m,VIIRS_NOAA20_CorrectedReflectance_TrueColor(hidden),VIIRS_SNPP_CorrectedReflectance_TrueColor,MODIS_Aqua_CorrectedReflectance_TrueColor(hidden),MODIS_Terra_CorrectedReflectance_TrueColor(hidden)&lg=true&t=2023-12-05-T13%3A57%3A16Z).   You won't need to create a new Renderable, but adding a new layer is definitely possible if you find a better layer to show the particular storm.    Again on Worldviewer you can quickly browse through the available layers and then add them following the [guide](https://docs.openspaceproject.com/en/latest/manual/content/globebrowsing/creation/readable-datasets.html) or looking at some of the examples, like in `data/assets/scene/solarsystem/planets/earth/layers/colorlayers/viirs_snpp_temporal.asset`.  The parameters to the `createTemporalGibsGdalXml` you can find from the [GIBS](https://nasa-gibs.github.io/gibs-api-docs/available-visualizations/) webpage


Your suggestion will definitely work;  there might be an alternative that could result in a smoother image by having the camera stay still over the Earth as time progresses smoothly.  To achieve that:
1. Increase the camera "capture" distance.  Settings -> Navigation Handler -> Orbital Navigator -> Follow Anchor Node Rotation Distance to a higher value
2. Distable all other nodes that you don't want visible. In particular the trails, the "Stars" and the "Milky Way Sphere" as those will be distracting when animating time
3. Enable the /Solar System/Earth/Earth global illumination action to turn off shadows on Earth (optional step, depending on the visuals you want to create)
4. Start a Session Recording
5. Set the delta time to something like 12h/s or so
6. End the session recording when you are done
7. Output the session recording as frames

**Reply by hn-88** [reply]
Thanks @alexanderbock - this is very useful. Will try it out.

**Reply by hn-88** [reply]
Looks like it's a bit complicated, due to so many data layers being available, and me being a novice in the GIS field. For example, I found daily data for 2013, Cyclone Phailin, at worldview using the water vapour aqua modis data, and using the corrected reflectance true colour base layer, I get a view approximating what I want. But unfortunately that satellite seems to have some gaps in coverage over the tropics/equator, leading to [ugly gaps in the imagery](https://worldview.earthdata.nasa.gov/?v=-20.060176062138524,-53.63290045378103,188.18884002582604,42.85944469692507&l=Reference_Labels_15m(hidden),Reference_Features_15m(hidden),Coastlines_15m,MODIS_Aqua_Water_Vapor_5km_Day(hidden),MODIS_Terra_CorrectedReflectance_TrueColor&lg=false&t=2013-10-11-T00%3A00%3A00Z).

**Reply by hn-88** [reply]
@alexanderbock  wrote in another thread, [here](https://github.com/OpenSpace/OpenSpace/discussions/3187#discussioncomment-9151593), 

> Yea, the Aqua and Terra satellites had a small enough field of view, that they would miss parts of the Earth on each pass around :/
> 
> Check if ESA's Copernicus satellites have the correct data: https://browser.dataspace.copernicus.eu/

**Reply by hn-88** [reply]
@alexanderbock No, unfortunately Copernicus data browser shows data only from 2016. So Cyclone Phailin of 2013 and Katrina of 2005 are not available there. INSAT and LANDSAT would have older data, but via https://dataspace.copernicus.eu/complementary-data only the Europe data is made available. 

Eg. https://www.mosdac.gov.in/gsmap-isro-rain shows data available from 2000 to 2022. But probably this is not image data (anyway it needs a sign-in) and probably it is NC4 netCDF or something like that. 

https://landsat.visibleearth.nasa.gov/view.php?id=15419 shows a close-up of Katrina damage, but I'm not sure how or where or if I can get a full globe image of the storm's progression.
